myosuite/envs/myo/__init__.py

```diff
@@ -298,6 +298,16 @@ def register_env_with_variants(id, entry_point, max_episode_steps, kwargs):
         }
     )
 
+# Gait Torso DEPRL Baseline ==============================
```

Comment:
@P-Schumacher  -- Lets make this more genetic. More people will enjoy using this in different ways.